<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbHandlerWidget</class>
 <widget class="QWidget" name="DbHandlerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label4">
     <property name="text">
      <string>Server Type</string>
     </property>
     <property name="buddy">
      <cstring>comboType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboType"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelHostname">
     <property name="text">
      <string>Hostname</string>
     </property>
     <property name="buddy">
      <cstring>editHostname</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editHostname"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label3">
     <property name="text">
      <string>Port</string>
     </property>
     <property name="buddy">
      <cstring>spinPort</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinPort">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="specialValueText">
      <string>default</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label5">
     <property name="text">
      <string>Username</string>
     </property>
     <property name="buddy">
      <cstring>editUsername</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="editUsername"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label6">
     <property name="text">
      <string>Password</string>
     </property>
     <property name="buddy">
      <cstring>editPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="editPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelDatabase">
     <property name="text">
      <string>Database</string>
     </property>
     <property name="buddy">
      <cstring>editDatabase</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLineEdit" name="editDatabase"/>
     </item>
     <item>
      <widget class="QToolButton" name="buttonSelectFile">
       <property name="text">
        <string>Select File</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/diropen.png</normaloff>:/img/diropen.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnTest">
       <property name="text">
        <string>Test Connection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
